tests/test_openjdk.py

```diff
@@ -75,7 +75,7 @@
             OPENJDK_21_CONTAINER, "21", marks=OPENJDK_21_CONTAINER.marks
         ),
         pytest.param(
-            OPENJDK_22_CONTAINER, "22", marks=OPENJDK_22_CONTAINER.marks
+            OPENJDK_23_CONTAINER, "23", marks=OPENJDK_23_CONTAINER.marks
```

Comment:
it seems this causes tests to break.

`'openjdk_23' not found in `markers` configuration option`

Reply:
yes, fixed now.